packages/cactus-plugin-ledger-connector-iroha2/src/main/typescript/web-services/transact-endpoint.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * ExpressJS `transact` endpoint
+ */
+
+import type { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+  IAsyncProvider,
+} from "@hyperledger/cactus-common";
+import {
+  IEndpointAuthzOptions,
+  IExpressRequestHandler,
+  IWebServiceEndpoint,
+} from "@hyperledger/cactus-core-api";
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginLedgerConnectorIroha2 } from "../plugin-ledger-connector-iroha2";
+import { safeStringifyException } from "../utils";
+
+import OAS from "../../json/openapi.json";
+
+export interface ITransactEndpointOptions {
+  logLevel?: LogLevelDesc;
+  connector: PluginLedgerConnectorIroha2;
+}
+
+export class TransactEndpoint implements IWebServiceEndpoint {
```

Comment:
@outSH Please make sure to include the Iroha version and the Cacti version in the name (it's not done consistently elsewhere in the code either so apologies for asking for this additional clarity).

Reply:
@petermetz Done.

Reply:
@outSH Thank you!